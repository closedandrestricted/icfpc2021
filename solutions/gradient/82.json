{"vertices":[[62,26],[48,42],[60,45],[89,59],[64,56],[77,43],[11,141],[58,76],[101,76],[77,64],[100,70],[28,126],[17,118],[99,60],[76,85],[81,82],[61,133],[22,114],[94,94],[80,104],[28,90],[39,104],[102,92],[41,104],[18,92],[36,116],[62,163],[87,128],[58,90],[100,73],[121,127],[127,117],[16,114],[102,105],[4,84],[38,80],[60,133],[114,85],[154,96],[37,103],[133,145],[26,94],[54,101],[35,39],[93,128],[33,133],[155,129],[76,110],[108,145],[155,65],[17,119],[63,100],[76,84],[39,115],[48,71],[40,9],[34,161],[148,96],[156,159],[132,163],[6,149],[48,74],[49,37],[25,52],[24,25]]}