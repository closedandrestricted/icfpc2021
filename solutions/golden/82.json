{"vertices":[[61,28],[47,44],[65,46],[97,70],[58,60],[78,59],[7,148],[77,61],[116,77],[85,77],[92,91],[31,138],[21,127],[104,88],[98,59],[73,85],[114,122],[24,123],[91,77],[92,96],[8,91],[42,117],[75,108],[12,125],[26,101],[36,130],[87,128],[87,121],[22,133],[91,97],[132,121],[130,113],[18,119],[109,109],[34,94],[19,110],[61,118],[106,86],[152,88],[41,120],[144,139],[42,119],[43,95],[35,46],[112,133],[51,149],[163,119],[90,117],[125,154],[156,57],[57,140],[57,103],[71,99],[68,118],[60,70],[29,15],[59,176],[150,88],[165,149],[151,169],[34,161],[30,98],[42,41],[31,66],[14,45]]}